src/main/scala/se/nimsa/sbx/dicom/DicomUtil.scala

```diff
@@ -243,4 +244,8 @@ object DicomUtil {
     if (name == null) "" else name
   }
 
+  def padToEvenLength(bytes: ByteString): ByteString = {
+    val padding = if (bytes.length % 2 != 0) ByteString(0) else ByteString.empty
```

Comment:
This does not seem to take into account that different VRs shall be padded with different bytes (typically space char or 0)